galter_subjects_utils/deltor.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
```

Comment:
Moved from converter.py to its own module since more specific than just a conversion.